plugins/woocommerce/includes/emails/class-wc-email-customer-new-account.php

```diff
@@ -131,6 +139,7 @@ public function get_content_html() {
 					'sent_to_admin'      => false,
 					'plain_text'         => false,
 					'email'              => $this,
+					'set_password_url'   => $this->set_password_url,
```

Comment:
That's a good point. I assumed it would be fine, because in the same place we use `$this->user_login` and `$this->user_pass` that are set in the same place (the `trigger` method in the same class).

Reply:
Do you think that's sufficient guarantee or shall we switch those to getters as well?

Reply:
This is a public method, so we don't know when it could be called. That said, I'm approving because your explanation makes sense and should have been set in trigger class.